ajax/libs/victor/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "victor",
+  "description": "A JavaScript 2D vector class with methods for common vector operations",
+  "keywords": [
+    "vector",
+    "2d"
+  ],
+  "filename": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/maxkueng/victor.git"
+  },
+  "homepage": "http://victorjs.org/",
+  "author": "Max Kueng <me@maxkueng.com> (http://maxkueng.com/)",
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "https://github.com/maxkueng/victor/raw/master/LICENSE"
+    }
+  ],
+  "npmName": "victor",
```

Comment:
Let's use git auto-update instead. Because the author only build and publish the files for browser on GitHub repo.
It would be like
```javascript
"autoupdate": {
    "source": "git",
    "target": "git://github.com/maxkueng/victor.git",
    "fileMap": [
      {
        "basePath": "build",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```

`files` is a pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer). `**/*` is for recursively fetching the files under `build`.
For more details about auto-update and `file` pattern, you can see [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)